packages/core-flows/src/order/workflows/create-orders.ts

```diff
@@ -0,0 +1,203 @@
+import { CreateOrderDTO, OrderDTO } from "@medusajs/types"
+import { MathBN, MedusaError } from "@medusajs/utils"
+import {
+  WorkflowData,
+  createWorkflow,
+  parallelize,
+  transform,
+} from "@medusajs/workflows-sdk"
+import { useRemoteQueryStep } from "../../common"
+import {
+  confirmInventoryStep,
+  findOneOrAnyRegionStep,
+  findOrCreateCustomerStep,
+  findSalesChannelStep,
+  getVariantPriceSetsStep,
+  getVariantsStep,
+  validateVariantsExistStep,
+} from "../../definition/cart"
+import { prepareConfirmInventoryInput } from "../../definition/cart/utils/prepare-confirm-inventory-input"
+import { prepareLineItemData } from "../../definition/cart/utils/prepare-line-item-data"
+import { createOrdersStep, updateOrderTaxLinesStep } from "../steps"
+import { prepareCustomLineItemData } from "../utils/prepare-custom-line-item-data"
+
+export const createOrdersWorkflowId = "create-orders"
+export const createOrdersWorkflow = createWorkflow(
+  createOrdersWorkflowId,
+  (input: WorkflowData<CreateOrderDTO>): WorkflowData<OrderDTO> => {
```

Comment:
discussion: I believe that we usually export an interface from the types package for the input, my thinking is that if in the future people add steps, they might ask for more data, and they should be able to use the current interface and expand it. wdyt? not necessarely now but just throwing it here